libsolidity/ast/AST.h

```diff
@@ -180,10 +183,12 @@ class SourceUnit: public ASTNode, public ScopeOpener
 
 	/// @returns a set of referenced SourceUnits. Recursively if @a _recurse is true.
 	std::set<SourceUnit const*> referencedSourceUnits(bool _recurse = false, std::set<SourceUnit const*> _skipList = std::set<SourceUnit const*>()) const;
+	bool experimentalParsingEnabled() const { return m_experimentalParsingEnabled; }
 
 private:
 	std::optional<std::string> m_licenseString;
 	std::vector<ASTPointer<ASTNode>> m_nodes;
+	bool m_experimentalParsingEnabled;
```

Comment:
```suggestion
	bool m_experimentalParsingEnabled = false;
```
I know that it's not necessary and will have zero effect, but it gives me peace of mind to just *always* do that on POD members.